builders/tools/explorers/verify-contract.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Verify Smart Contracts
+description: Learn about how to verify smart contracts on Moonbeam networks using one of the available block explorers. 
+
+---
+# Verify Smart Contracts
+
+![Explorer Banner](/images/builders/tools/explorers/verify-contract/verify-contract-banner.png)
+
+## Introduction {: #introduction } 
+
+Verifying smart contracts on a block explorer is great way of improving the transparency and security of deployed smart contracts on Moonbeam. Users can directly view the source code for verified smart contracts, and for some block explorers, can also directly interact with the contract's public methods through the block explorer's interface. 
+
+This page will outline the steps for verifying smart contracts on Moonbeam networks.
+
+## Deploying the Contract {: #deploying-the-contract }
+
+In order to verify a smart contract on a block explorer, the contract must be first deployed on the target network first. For this tutorial, we will be deploying the smart contract to [Moonbase Alpha](/builders/get-started/moonbase/). 
```

Comment:
We avoid using 2nd person like "we". In this case you can do "This tutorial will deploy a basic smart contract to..."